docs/conf.py

```diff
@@ -23,7 +23,7 @@
 author = "Nathan Horrigan"
 
 # The full version, including alpha/beta/rc tags
-release = "0.3.0"
+release = "0.5.1"
```

Comment:
nice catch. best to get the version from the module itself. Wagtail does https://github.com/wagtail/wagtail/blob/master/docs/conf.py#L80-L86

Reply:
@zerolab I pushed [this commit](https://github.com/GrappleGQL/wagtail-grapple/pull/91/commits/c631764d2ac6b1a65203a6d954db8e6ccb93d13c). Not sure if this is the best way 🙂 .